src/core/qgis.h

```diff
@@ -2919,7 +2919,7 @@ class CORE_EXPORT Qgis
      */
     enum class AreaUnit SIP_MONKEYPATCH_SCOPEENUM_UNNEST( QgsUnitTypes, AreaUnit ) : int
       {
-      SquareMeters = 0 SIP_MONKEYPATCH_COMPAT_NAME( AreaSquareMeters ), //!< Square meters
```

Comment:
that would have deserved a fix in sipify 😛

Reply:
I tried, and failed :laughing: